### ISSUE TITLE
ASIO plugin - Issue with Channel 4

### ISSUE THREAD

**Issue by streamia-livestreaming** [Andersama/obs-asio#107]
**Windows** 10 - **OBS** 27.1.3 64 bit - **ASIO** plugin version 3.1.0 - **Audio device** Yamaha Steinberg UR44C USB

**The problem**:
Channel No 4 is not recorded nor streamed.

**Description**:
I tested this issue in several ways, getting into the same conclusion.
Used 4.0, 5.1 or 7.1 multichannel audio. Although, in all cases, I see a volume indicator for channel 4 on audio mixer, this specific channel is not monitored, recorded or streamed.
This was not an issue or prior version for OBS 26-.

**Comments**:
I realized this when lately streamed to "**AWS Media Live/Packager**" and tried to remix audio for HLS multitrack output, redirecting each source channel to a separate track.

Please ask for any details regarding the issue.

Regards.

**Comment by Andersama** [comment]
Major revisions of obs can break abi compatibility with older plugins. That sounds like what's happening here, download the latest release for v27, see if that helps.

**Comment by streamia-livestreaming** [comment]
> ...download the latest release for v27, see if that helps.

It is the latest release installed, v. 27.1.3.
Did your own verify this issue?

Regards, Panagiotis.

**Comment by Andersama** [comment]
Yeah sorry didn't double check, the description reads exactly like an abi break, ages ago the volume meter didn't display meters correctly, can't remember if that was fixed. If you're saying between v26 and v27 the plugin stopped working, that's going to be obs at fault, there shouldn't be a significant functional change to the plugin, assuming you're downloading the ones built with the same library.
